api/src/main/java/au/org/aodn/nrmn/restapi/validation/validators/formatted/TotalCheckSum.java

```diff
@@ -0,0 +1,37 @@
+package au.org.aodn.nrmn.restapi.validation.validators.formatted;
+
+import au.org.aodn.nrmn.restapi.model.db.StagedRowError;
+import au.org.aodn.nrmn.restapi.model.db.composedID.ErrorID;
+import au.org.aodn.nrmn.restapi.model.db.enums.ValidationCategory;
+import au.org.aodn.nrmn.restapi.model.db.enums.ValidationLevel;
+import au.org.aodn.nrmn.restapi.validation.BaseFormattedValidator;
+import au.org.aodn.nrmn.restapi.validation.StagedRowFormatted;
+import cyclops.control.Validated;
+import lombok.val;
+
+import java.util.Map;
+
+public class TotalCheckSum extends BaseFormattedValidator {
```

Comment:
This one isn't included for RLS or ATRC?

Reply:
I have added back the check as "Warning" but i had a discussion with Bene where this check was not usefull since total is macro generated. I will confirm tomorrow if i can remove it